storage/rocksdb/ha_rocksdb.cc

```diff
@@ -920,6 +920,7 @@ static long long rocksdb_compaction_sequential_deletes_file_size = 0l;
 static uint32_t rocksdb_validate_tables = 1;
 char *rocksdb_datadir;
 static uint32_t rocksdb_max_bottom_pri_background_compactions = 0;
+static int32_t rocksdb_block_cache_numshardbits = -1;
```

Comment:
```suggestion
static int rocksdb_block_cache_numshardbits = -1;
```
`MYSQL_SYSVAR_INT`, `HyperClockCacheOptions`, `NewLRUCache` all expect an `int` here

Reply:
Updated the PR to fix this